Resources/Prototypes/DeltaV/Roles/Jobs/Command/admin_assistant.yml

```diff
@@ -0,0 +1,32 @@
+- type: job
+  id: AdministrativeAssistant
+  name: job-name-admin-assistant
+  description: job-description-admin-assistant
+  playTimeTracker: JobAdminAssistant
+  requirements:
+    - !type:DepartmentTimeRequirement
+      department: Command
+      time: 21600 # 6 hours
+    - !type:OverallPlaytimeRequirement 
+      time: 36000 # 10 hours
+    - !type:WhitelistRequirement
```

Comment:
If RP-focused roles like HoP and Lawyer aren't whitelisted, then this shouldn't be either. No particular danger associated with this role if they have command hours

Reply:
I think it's safe to introduce new roles via whitelist and then expand them out to other players but :shrug:

Reply:
Took out the whitelist for now